src/protocols/secure_channel/CASEServer.cpp

```diff
@@ -181,4 +179,34 @@ void CASEServer::OnSessionEstablished(const SessionHandle & session)
                     ChipLogValueScopedNodeId(session->GetPeer()));
     PrepareForSessionEstablishment(session->GetPeer());
 }
+
+void CASEServer::SendBusyStatusReport(Messaging::ExchangeContext * ec)
+{
+    using namespace Protocols::SecureChannel;
+
+    // 500 milliseconds is an example value mentioned in specification section "4.10.1.5. Busy"
+    uint16_t minWaitTime    = 500; // milliseconds
+    auto protocolDataHandle = System::PacketBufferHandle::NewWithData(&minWaitTime, sizeof(minWaitTime));
```

Comment:
This is not the right data.  Spec says: 

> Set the StatusReport ProtocolData to a 16-bit (two byte) little-endian value indicating the minimum time in milliseconds

This is not doing little-endian.  Please use allocate the packet buffer with the right size and no data and write to it with LittleEndian::PacketBufferWriter.

Also, why are we allocating a packet buffer with a reserve here?  We should allocate it without reserve; we don't need the extra reserve memory here.

Reply:
Since the StatusReport was also doing the little endian encoding I though this could work. But, yeah but the spec comment is specifically says about encoding protocol data as little endian. Fixed!